validator/node/node.go

```diff
@@ -105,7 +105,7 @@ func NewValidatorClient(cliCtx *cli.Context) (*ValidatorClient, error) {
 		return nil, err
 	}
 	if len(pubKeys) == 0 {
-		log.Warn("No keys found; nothing to validate")
+		log.Error("No keys found. Please verify `--keystore-path` is the correct path")
```

Comment:
Why does the validator continue running? Also, I think this flag is only relevant for accounts V1.

Reply:
> Why does the validator continue running? 
That's how it is today. Changing the underly behavior is outside of the scope.

> Also, I think this flag is only relevant for accounts V1
That's fine. We still should fix it